### PR TITLE
Python sphinxcontrib-newsfeed-0.1.4

### PR DESCRIPTION
**Pull request by massimo-zaniboni**
###### Motivation for this change

Added a Python package.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -24571,6 +24571,22 @@ in {
     };
   });
 
+  sphinxcontrib_newsfeed = buildPythonPackage (rec {
+    name = "sphinxcontrib-newsfeed-${version}";
+    version = "0.1.4";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/2b/5e/8bc839b5c4ef030bf26eede24208a49f25d00033cbd4969b3895264f14db/${name}.tar.gz";
```
Comment (lsix) [inline pkgs/top-level/python-packages.nix]: You could prefer

```
url = "mirror://pypi/s/sphinxcontrib-newsfeed/${name}.tar.gz";
```

With this form, future updates are simpler (`version` and `sha256` have to be updated, not the `url`).

**Comment by lsix** [comment]
Hi,

Thanks for the changes. Could you also [squash](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) the two commits into a single one before we incorporate it into master?

**Comment by FRidh** [comment]
Squash locally and force push the new commit.

On Thu, Dec 29, 2016 at 11:14 AM, Massimo Zaniboni <notifications@github.com

**Comment by lsix** [comment]
@massimo-zaniboni could you also update the commit message to fit the [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) guidelines? It should be `pythonPackages.sphinxcontrib_newsfeed: init at 0.1.4`. I am sorry I did not see that earlier.

A simple `git commit --amend` and then `git push -f …` should be just fine.

**Comment by massimo-zaniboni** [comment]
@Isix obviously my fault: the CONTRIBUTING.md file is short, and also in the proposed checklist! :-)

**Pull request merged**